gradle.properties

```diff
@@ -1,4 +1,4 @@
-releaseVersion = 0.3.1-SNAPSHOT
+releaseVersion = 0.3.1
```

Comment:
I wonder if we should go with 0.4.0, since we're changing the structure of the emitted metrics